### ISSUE TITLE
installation of package ‘clpAPI’ had non-zero exit status

### ISSUE THREAD

**Issue by pnavaro** [otrecoding/OTrecod#6]
R CMD check failed on linux

```
configure: error: Could not find Clp_C_Interface.h:
6598      clpAPI requires clp from http://www.coin-or.org/projects/Clp.xml
6599      use --with-clp-include or CLP_INCLUDE to specify the include path.
6600ERROR: configuration failed for package ‘clpAPI’
6601* removing ‘/home/travis/R/Library/clpAPI’
6602Error: Failed to install 'OTrecod' from GitHub:
6603  (converted from warning) installation of package ‘clpAPI’ had non-zero exit status
6604Execution halted
6605The command "Rscript -e 'remotes::install_github(c("otrecoding/OTrecod"))'" failed and exited with 1 during .
```

**Comment by pnavaro** [comment]
Pour vérifier que ca passe il faut aller ici https://travis-ci.org/github/otrecoding/OTrecod

**Comment by pnavaro** [comment]
fix by abf3c5471cedea515492368358ca76d7aaf8370e
